zip-####.rst

```diff
@@ -50,37 +51,82 @@ Code (Title 26 of the U.S. Code). [#section501c3]_
 "Community Advisory Panel" refers to the panel of community members assembled
 by the Zcash Foundation and described at [#zf-community]_.
 
+"Zcash Community Grants", also called "ZCG", refers to grants program
+(formerly known as ZOMG) that funds independent teams entering the Zcash ecosystem, 
+to perform major ongoing development (or other work) 
+for the public good of the Zcash ecosystem.
+<https://zcashcommunitygrants.org/>
+
+"Financial Privacy Fund", also called "FPF" refers to the start-up non-profit
+organization incorporated and based in the Cayman Islands.
+<https://www.financialprivacyfoundation.org/>
+
+"Qedit" refers to the company founded in 2016 by a world-class team of 
+accomplished tech entrepreneurs, researchers, and developers; 
+QEDIT has emerged as a global leader in the field of Zero-Knowledge Proofs.
+<https://qed-it.com/about-us/>
+
+"ZecHub" refers to the team of content creators who have supported Zcash
+through a series of ZCG approved grants.
+<https://forum.zcashcommunity.com/t/zechub-an-education-hub-for-zcash-2024-continued/47947>
+
 The terms "Testnet" and "Mainnet" are to be interpreted as described in
 section 3.12 of the Zcash Protocol Specification [#protocol-networks]_.
 
+The term "ZSA" is to be interpreted as "Zcash Shielded Assets" the protocol
+feature enhancement (and subsequent application layer, legal, maintenance 
+efforts, et al) on-going by Qedit, et al.
+<https://forum.zcashcommunity.com/t/grant-update-zcash-shielded-assets-monthly-updates/41153>
+
 
 Abstract
 ========
 
 This proposal describes a structure for the Zcash Development Fund, to be
-enacted in Network Upgrade 4 and last for 4 years. This Dev Fund would consist
-of 20% of the block subsidies, split into 3 slices:
+enacted in Network Upgrade 6 and last for 4 years. This Dev Fund would consist
+of 15% of the block subsidies, as the following allocations:
+
+* 5% for the Zcash Foundation (for internal work and grants);
+* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
+and contingent upon an ecosystem mandated "non-direct funding model" for the future;
```

Comment:
This is not clear to me - if a non-direct funding model is proposed, how will that proposal interact with this ZIP?